ui/src/app/edge/history/timeofusetariff/chart.component.ts

```diff
@@ -138,8 +138,8 @@ export class TimeOfUseTariffDischargeChartComponent extends AbstractHistoryChart
             borderColor: 'rgba(0,0,200,0.9)'
           });
 
-          // Show charge data only for the new controller.
-          if (this.component.factoryId === 'Controller.Ess.Time-Of-Use-Tariff') {
+          // Show data based on control modes selected.
+          if (this.component.properties.controlMode === 'CHARGE_CONSUMPTION') {
```

Comment:
a few lines down here, there is still some dead code laying around

Reply:
It was deliberate to show values in future. Once we have better prediction.

Reply:
i remember, then ok